### PR TITLE
Added weightedCorrelationNetworkAnalysis.json

### PR DESCRIPTION
**Pull request by MaaikevS**
Related to #139

### CODE REVIEW

File: instances/technique/weightedCorrelationNetworkAnalysis.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/WeightedCorrelationNetworkAnalysis",
```
Comment (lzehl) [inline instances/technique/weightedCorrelationNetworkAnalysis.json]: `.../weightedCorrelationNetworkAnalysis`

**Review by lzehl — CHANGES_REQUESTED** [review]

**Comment by MaaikevS** [comment]
@lzehl I see now that this PR was never merged (but apparently I closed it). Would you be able to merge it?

**Comment by lzehl** [comment]
@MaaikevS could you have a look at the editions I made. If you confirm I would merge the PR

**Comment by MaaikevS** [comment]
Yup, I am happy with the edits. Thanks!

**Comment by lzehl** [comment]
@MaaikevS and merged :) thank you!

**Pull request merged**